src/libraries/System.Security.Cryptography/src/System.Security.Cryptography.csproj

```diff
@@ -0,0 +1,45 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
```

Comment:
I don't think this one is a partial facade assembly, right?

Reply:
Hmmm... right.  Primitives was because of the CryptographicException typeforward.  I should be able to remove that here.